nym-wallet/src/components/Accounts/modals/ConfirmPasswordModal.tsx

```diff
@@ -14,28 +14,32 @@ export const ConfirmPasswordModal = ({
   onConfirm: (password: string) => Promise<void>;
 }) => {
   const { isLoading, error } = useContext(AccountsContext);
+  const theme = useTheme();
 
   return (
     <Dialog
       open={Boolean(accountName)}
       onClose={onClose}
       fullWidth
-      PaperComponent={Paper}
-      PaperProps={{ elevation: 0 }}
+      PaperProps={{
+        style: { border: `1px solid ${theme.palette.nym.nymWallet.modal.border}` },
+      }}
     >
-      <DialogTitle>
-        <Typography variant="h6">Switch account</Typography>
-        <Typography fontSize="small" sx={{ color: 'grey.600' }}>
-          Confirm password
-        </Typography>
-      </DialogTitle>
-      <ConfirmPassword
-        onConfirm={onConfirm}
-        error={error}
-        isLoading={isLoading}
-        buttonTitle="Switch account"
-        onCancel={onClose}
-      />
+      <Paper>
```

Comment:
I think, MUI `Dialog` already use a `Paper` by default to render the body. I suggest using a `Box` here instead, to avoid rendering two paper, one nested in another one.